src/SmartComponents/DriftPage/DriftPage.js

```diff
@@ -1,42 +1,89 @@
     }
 
     render() {
-        return (
-            <React.Fragment>
-                <PageHeader>
-                    <PageHeaderTitle title='System Comparison'/>
-                </PageHeader>
-                <Main className="drift">
-                    <Card className='pf-t-light pf-m-opaque-100'>
-                        <CardBody>
-                            <Grid className='drift-toolbar'>
-                                <GridItem span={ 4 }>
-                                    <SearchBar />
-                                </GridItem>
-                                <GridItem span={ 2 }>
-                                    <FilterDropDown />
-                                </GridItem>
-                            </Grid>
-                            <div>
-                                <DriftTable />
-                            </div>
-                        </CardBody>
-                    </Card>
-                </Main>
-            </React.Fragment>
-        );
+        if (this.props.compare.facts) {
+            return (
+                <React.Fragment>
+                    <PageHeader>
+                        <PageHeaderTitle title='System Comparison'/>
+                    </PageHeader>
+                    <Main className="drift">
+                        <Card className='pf-t-light pf-m-opaque-100'>
+                            <CardBody>
+                                <Grid className='drift-toolbar'>
+                                    <GridItem span={ 4 }>
+                                        <SearchBar />
+                                    </GridItem>
+                                    <GridItem span={ 2 }>
+                                        <FilterDropDown />
+                                    </GridItem>
+                                </Grid>
+                                <div>
+                                    <DriftTable />
+                                </div>
+                            </CardBody>
+                        </Card>
+                    </Main>
+                </React.Fragment>
+            );
+        }
+
+        else {
+            return (
+                <React.Fragment>
+                    <PageHeader>
+                        <PageHeaderTitle title='System Comparison'/>
+                    </PageHeader>
+                    <Main className="drift">
+                        <Card className='pf-t-light pf-m-opaque-100'>
+                            <CardBody>
+                                <div>
+                                    <center>
+                                        { /*<RenderPage { ...this.props.compare } />*/ }
+                                        <EmptyTableState />
+                                    </center>
+                                </div>
+                            </CardBody>
+                        </Card>
+                    </Main>
+                </React.Fragment>
+            );
+        }
```

Comment:
There's a lot of overlap here. Can we reduce the if/else to just the parts that are changing?

Reply:
I'm struggling with the best way to handle this then. You can't put if/else statements inside of JSX (inside of the return). Conditional rendering is handled like so: https://reactjs.org/docs/conditional-rendering.html but we're looking at rending 1 component vs. 3 components, so that complicates things.

I'm open to thoughts on this.

Reply:
Only other thought I have is to introduce two new components:

 * DriftEmptyStatePage
 * DriftTablePage

That could be overkill for now.